tools/topology/CMakeLists.txt

```diff
@@ -117,6 +117,7 @@ set(TPLGS
 	"sof-cml-rt1011-rt5682\;sof-cml-rt1011-rt5682\;-DPLATFORM=cml\;-DPPROC=volume"
 	"sof-cml-rt1011-rt5682\;sof-cml-eq-rt1011-rt5682\;-DPLATFORM=cml\;-DPPROC=eq-iir-eq-fir-volume"
 	"sof-tgl-nocodec\;sof-tgl-nocodec"
+	"sof-tgl-nocodec-ci\;sof-tgl-nocodec-ci"
```

Comment:
the name is really terrible. something like sof-tgl-nocodec-fw-eref.ri would be clearer, no?

Reply:
@plbossart  Actually this tplg will include both fw echo reference and hw echo reference